docs/README.md

```diff
@@ -140,14 +101,27 @@ env:
 
 Check out [Kubernetes documentation](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/) for more information about injecting environment variables into applications!
 
-### Run as Non-Root
```

Comment:
maybe add this "Run as non-root user" back in?

Reply:
Added it back in at the top of the page